vulnfeeds/cves/versions.go

```diff
@@ -611,7 +611,16 @@ func ValidateAndCanonicalizeLink(link string) (canonicalLink string, err error)
 	}
 	backoff := retry.NewExponential(1 * time.Second)
 	if err := retry.Do(context.Background(), retry.WithMaxRetries(3, backoff), func(ctx context.Context) error {
-		resp, err := http.Head(link)
+		req, err := http.NewRequest("HEAD", link, nil)
+		if err != nil {
+			return err
+		}
+
+		// security.alpinelinux.org responds with text/html content.
```

Comment:
```suggestion
		// security.alpinelinux.org responds with text/html content.
		// default HEAD request in Go does not provide any Accept headers, causing a 406 response
```

Added some clarification for myself.